<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI_StreamPreferences</class>
 <widget class="QWidget" name="GUI_StreamPreferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>283</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="cbUpdateMetadata">
     <property name="text">
      <string>Update metadata (can be set individually radio stations)</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="cbShowHistory">
     <property name="text">
      <string>Show history (only works if metadata updates are enabled for a radio station)</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labBufferSize">
     <property name="text">
      <string>Buffer size</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="sbBufferSize">
     <property name="suffix">
      <string> ms</string>
     </property>
     <property name="minimum">
      <number>20</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="value">
      <number>500</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QCheckBox" name="cbNewTab">
     <property name="text">
      <string>Open Streams in new tab</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="cbOpenInLockedPlaylist">
     <property name="text">
      <string>Open streams and podcasts in locked playlist</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cbNewTab</tabstop>
  <tabstop>cbShowHistory</tabstop>
  <tabstop>sbBufferSize</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
